modules/caching-materials-manager-browser/test/caching_materials_manager_browser.test.ts

```diff
@@ -16,7 +16,7 @@
 /* eslint-env mocha */
 
 import { expect } from 'chai'
-import 'mocha'
+// import 'mocha'
```

Comment:
In most other files, this import was deleted. In this file, it was commented out.